io/input/inputformat_store.cpp

```diff
@@ -17,113 +17,191 @@
 #include <string>
 #include <unordered_map>
 
-#include "base/assert.hpp"
 #include "base/session_local.hpp"
 
 namespace husky {
 namespace io {
 
-thread_local int g_gen_inputformat_id = 0;
-thread_local InputFormatMap* InputFormatStore::s_inputformat_map = nullptr;
+thread_local int InputFormatStore::default_inputformat_id = 0;
```

Comment:
You turned this file to a older version. Please undo this. That's why the testing on travis ci is failed.